repository/src/main/java/org/pentaho/platform/repository2/unified/fs/FileSystemRepositoryFileDao.java

```diff
@@ -225,7 +228,10 @@ public RepositoryFile internalGetFile( File f ) {
       String jcrPath = f.getAbsolutePath().substring( rootDir.getAbsolutePath().length() );
       if ( jcrPath.length() == 0 ) {
         jcrPath = "/";
+      } else if ( IS_WINDOWS ) {
+        jcrPath = FilenameUtils.separatorsToUnix( jcrPath );
```

Comment:
This won't work for our purposes, an *escaped* path that looks like \\home\\admin\\file.wcdf, gets transformed into //home//admin//file.wcdf which is not what was intended.